Dockerfile

```diff
@@ -46,17 +46,29 @@ RUN pip3 install -r server-requirements.txt
 COPY build.sh .
 RUN sh build.sh
 
+RUN  apt update
```

Comment:
Let's put this into a single `RUN` command to minimize the layers! Housecleaning type stuff